cpp/msbuild/ice.test.sln

```diff
@@ -1160,30 +1160,69 @@ EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "types", "types", "{2618669B-54B1-4013-B6EC-C1389391E3AA}"
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "writer", "..\test\DataStorm\api\msbuild\writer\writer.vcxproj", "{85F74421-2B78-448B-91F0-496526EB365F}"
```

Comment:
DataStorm tests project were missing a dependency on TestCommon, as a result the parallel build can fail.